### DISCUSSION TITLE
Change default emulator for GBA

### DISCUSSION THREAD

**Discussion by kirkegaard** [OnionUI/Onion#487] [Q&A]
Is there a way to use mGBA instead of the default emulator in the Console menu?
I know i can install it and go the Expert menu but i would rather use the Console menu and disable the expert menu :)

**Comment by dwmccoy** [comment]
edit the launch.sh file located in the following location sdcard/EMU/GBA/ and edit the following line from this 

HOME=/mnt/SDCARD/RetroArch/ $progdir/../../RetroArch/retroarch -v -L $progdir/../../RetroArch/.retroarch/cores/**gpsp_libretro.so** "$1"

to this 

HOME=/mnt/SDCARD/RetroArch/ $progdir/../../RetroArch/retroarch -v -L $progdir/../../RetroArch/.retroarch/cores/**mgba_libretro.so** "$1"

Keep in mind any Onion OS updates will overwrite this file back to defaults and revert back to the default emulator.
